zndraw/static/World/World.js

```diff
@@ -232,6 +233,12 @@ class World {
    */
   start() {
     loop.start();
+    // center camera if there are particles
+    const particles = this.particles.cache.get(0);
+    if (particles === undefined || particles.length === 0) {
+    } else {
+      centerCamera(this.selection.controls, this.particles);
+    }
```

Comment:
I'd prefer the loop to be inverted to only `if ...` without the else, because `if` is empty.

Reply:
Makes sense.